cypress/integration/RichTextEditor.spec.ts

```diff
@@ -470,6 +532,30 @@ describe('Rich Text Editor', () => {
 
           expectRichTextFieldValue(expectedValue);
         });
+
+        it('should unwrap blockquote', () => {
+          editor().click().typeInSlate('some text');
+
+          getQuoteToolbarButton().click();
+
+          const expectedQuoteValue = doc(
+            block(BLOCKS.QUOTE, {}, block(BLOCKS.PARAGRAPH, {}, text('some text', []))),
+            block(BLOCKS.PARAGRAPH, {}, text('', []))
+          );
+          expectRichTextFieldValue(expectedQuoteValue);
```

Comment:
Do we really need to assert all of this in each of these tests or could we at least put it into a helper `insertTextAsBlockQuote('some text')` ?

Reply:
We have 3 elements that cannot interact with blockquote: headings, lists and HR. I think it makes sense to have test for each element interaction. I'm ok with the helper idea, don't mind.